docs/cli_commands.md

```diff
@@ -322,6 +322,12 @@ Creates a keymap.json from a keymap.c.
 qmk c2json -km KEYMAP -kb KEYBOARD [-q] [--no-cpp] [-o OUTPUT] filename
 ```
 
+**Examples**:
+
+```
+qmk c2json -km default -kb handwired/dactyl_promicro /home/my_user/qmk_firmware/keyboards/handwired/dactyl_promicro/keymaps/my_user/keymap.c
+```
```

Comment:
Since 22170, its somewhat cleaner to document the simplified versions as it avoids the error of mixing `default` and `my_user` as keymap names

````suggestion
```
qmk c2json -km default -kb handwired/dactyl_promicro
```

or with filename:

```
qmk c2json keyboards/handwired/dactyl_promicro/keymaps/default/keymap.c
```
````